tests/examples/80-columns.txt

```diff
@@ -0,0 +1 @@
+abcdefghigklmnopqrstuvxyzabcdefghigklmnopqrstuvxyzabcdefghigklmnopqrstuvxyzabcdefghigklmnopqrstuvxyz
```

Comment:
Shouldn't this file be called `100-columns.txt`? Another proposal: `long-single-line.txt`.

Reply:
Oh, I like `long-single-line.txt`, I'll go with that.